### PR TITLE
added deleting img from static

### PR DESCRIPTION
**Pull request by lestraza**

### CODE REVIEW

File: client/src/components/Auth/store/Auth.store.ts [hunk 1]
```diff
@@ -235,7 +236,16 @@ export class AuthStore extends AbstractStore {
                             this.client.avatarUrl = avatarUrl
                         })
                     })
-                    .catch()
+                    .catch((err) => {
```
Comment (stahlhammer88) [inline client/src/components/Auth/store/Auth.store.ts:239]: return the service call from .then and remove this .catch

File: server/index.js [hunk 2]
```diff
@@ -4,6 +4,10 @@ const mongoose = require('mongoose')
 const cookieParser = require('cookie-parser')
 const path = require('path')
 const uuid = require('uuid').v1
+const fs = require('fs')
+
+const { promisify } = require('util')
+const unlinkAsync = promisify(fs.unlink)
```
Comment (stahlhammer88) [inline server/index.js:10]: You never use promisify and unlinkAsync

File: client/src/components/Auth/store/Auth.store.ts (hunk 1 above)
Comment (lestraza) [reply client/src/components/Auth/store/Auth.store.ts]: ok

File: server/index.js (hunk 2 above)
Comment (lestraza) [reply server/index.js]: ok

File: client/src/components/Auth/store/Auth.store.ts [hunk 3]
```diff
@@ -223,19 +223,20 @@ export class AuthStore extends AbstractStore {
...
                     _id,
                 }
-                saveProfilePhotoReq(data)
+                return saveProfilePhotoReq(data)
                     .then(() => {
                         runInAction(() => {
                             this.client.avatarUrl = avatarUrl
                         })
                     })
-                    .catch()
```
Comment (stahlhammer88) [inline client/src/components/Auth/store/Auth.store.ts:238]: why have you removed both catches?

**Pull request merged**